modules/theme.css

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2023 LupLab
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+/*
+ * Body elements
+ */
+#theme-toggler {
+  width: 50px;
+  height: 25px;
+  border: 2px solid var(--bs-primary);
+  border-radius: 25px;
+}
+
+#theme-toggler:focus {
+  box-shadow: none;
+}

```

Comment:
One of the great things of bootstrap is that you hardly have to define your own CSS (unless really necessary). Let's drop this CSS file and try to integrate your visual stuff using standard bootstrap classes only (it's normal if it's doesn't match 100%, but you can get close). Thanks!

Reply:
@joel-porquet integrated with bootstrap classes. Please review

Reply:
That's great, thanks.

Now remember that one of the requested changes was that a pull request shouldn't have  any "fix/update/refactor" commits. The idea is that a pull request can be finely-tuned up until it gets merged into the main branch, so we want it to be perfect from the get go. This way, the project's history (in its main branch) is as clean as possible.

So here, you should blend this last commit into the first one. After that, we'll never know there was ever a CSS file! Look into `git rebase -i` to squash this commit into the first one. This is one of git's most powerful command, and you won't regret learning about it! :)

Let me know!

Reply:
That was quite a learning experience. Implemented it into one commit

Reply:
Thanks Sachin! Sorry for making you jump in all these hoops, I wanted to make sure you were up for the task (which you proved to be!). Being able to use git efficiently will certainly give an edge later in your career, and it will also be useful to work on this project anyway.

Alright, I think that's enough of a screening test. Let's meet and talk now! :)

Can you fill out a when2meet with your availability between tomorrow and the end of next week and we'll figure out a time that works? Thanks again and looking forward to discussing our collaboration!

Reply:
Of course Professor. I get why this whole testing process was required.

Here is the link to the when2meet as requested: https://www.when2meet.com/?26699596-i8cM1
I have sent this over email too